Formula/source-to-image.rb

```diff
@@ -0,0 +1,14 @@
+class SourceToImage < Formula
+  desc "Tool for building source and injecting into docker images"
+  homepage "https://github.com/openshift/source-to-image"
+  url "https://github.com/openshift/source-to-image/releases/download/v1.1.3/source-to-image-v1.1.3-ddb10f1-darwin-amd64.tar.gz"
```

Comment:
We want to have full control over what we build; we provide pre-built binaries ("bottles") so users who don’t want to build from source won’t have to install Go.

See https://github.com/Homebrew/brew/blob/master/docs/Acceptable-Formulae.md#we-dont-like-binary-formulae